package.json

```diff
@@ -9,7 +9,7 @@
     "lib": "./lib"
   },
   "dependencies": {
-    "lodash": "2.4.0"
+    "lodash": "4.5.1"
```

Comment:
None that I can see - I was just matching the version used by Revolver.  Bumping to 4.13.1 should be fine.